const networkError = 'Произошла сетевая ошибка, свяжитесь с технической поддержкой. ';
const internalError = 'Произошла внутренняя ошибка, свяжитесь с технической поддержкой. ';
const successRecord = 'Запись успешно создана, записи можно смотреть в ';

export {
  networkError,
  internalError,
  successRecord,
}
